api/python/quilt3/util.py

```diff
@@ -511,7 +509,7 @@ def validate_key(key):
         )
 
     for part in key.split('/'):
-        if part in ('', '.', '..'):
+        if part in ('.', '..'):
```

Comment:
why did you do that?
that seems wrong

Reply:
Because the (now valid) path '//foo' splits to ['', 'foo'], and causes an error.

To be fair, that was only an error for one of the two occurences. I will revert the other.

Reply:
My bad. Reverted.